crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -291,7 +291,7 @@ where
                     }
                     Err(err) => {
                         tx.send(Err(EngineApiError::Internal(Box::new(err)))).ok();
-                        return
+                        return;
```

Comment:
```suggestion
                        return
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/153